source/manual/encrypted-hiera-data.html.md

```diff
@@ -440,6 +440,18 @@ to fail when using GPG version 1 with lots of credentials.
 
 If you see this error:
 
+```
+keyserver receive failed: Network is unreachable
+```
+
+Run this command to add `standard-resolver` to `dirmngr.conf`
+
+```sh
+echo "standard-resolver" >> ~/.gnupg/dirmngr.conf
```

Comment:
It's possible that this settings change won't take effect because there may be a background process that doesn't reload its configuration. Config can be reloaded as follows:

```suggestion
echo "standard-resolver" >> ~/.gnupg/dirmngr.conf
gpgconf --kill all
```

Reply:
👍